cg/store/api/status.py

```diff
@@ -28,67 +28,6 @@
 class StatusHandler(BaseHandler):
     """Handles status states for entities in the database."""
 
-    def samples_to_receive(self, external=False) -> Query:
-        """Fetch incoming samples."""
-        return (
-            self.Sample.query.join(
-                Sample.application_version,
-                ApplicationVersion.application,
-            )
-            .filter(
```

Comment:
see comment above, these functions do indeed a join and a filter. It is better to split the join into a separate query function and then call the query here to apply the filters

Reply:
Fixed